app/models/curiosity_card.rb

```diff
@@ -24,6 +24,8 @@ class CuriosityCard < ApplicationRecord
       where(category: category).order('RANDOM()').take(5)
   }
 
+  scope :search, ->(text) { where('title LIKE ?', "%#{text}%") }
```

Comment:
el scope puede decidir que hacer
```suggestion
  scope :search, ->(text=nil) { text.nil? ? all : where('title LIKE ?', "%#{text}%") }
```